### DISCUSSION TITLE
Countdown to the 🪐 Universe: Actions Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#70010] [Actions]
![Screenshot 2023-10-09 at 7 11 05 PM](https://github.com/community/community/assets/95834130/001403df-3bb3-4d8e-84b3-6af911633176)
> Alt Text: This banner image shows a black background with a blue and green connecting square towards the right along with a GitHub logo at the bottom. 

GitHub has invested in a variety of different CI systems throughout its history. With each system, our aim has been to enhance the development experience for both GitHub engineers writing and deploying code and for engineers maintaining the systems.

With our older CI systems reaching scaling capacity, we’ve been making some breakthrough achievements not only to transition to a fully featured CI system, but also to develop, experience, and utilize the systems we are creating for all Actions users. 🚀 

If you’re a GitHub Team or GitHub Enterprise Cloud customer, [GitHub Actions larger runners](https://github.blog/2022-09-01-github-actions-introducing-the-new-larger-github-hosted-runners-beta/) are our commitment to you for delivering a superior developer experience. We will share another BIG change below, but before we dive into it, let’s get started by ✨sparkling some of our amazing community members who stood out in the Actions space. 

 ## 💌  From the Community - Shoutout 📢  to these members

- Leading by example - A round of applause 👏 for @elendil7 for a descriptive and well thought out response that helped @ash-dey-sa successfully [run a job on pull request merge to specific branch](https://github.com/orgs/community/discussions/67160). 
- @seyLu continues to be one of the top contributors ⭐ in the space.Though they have contributed to multiple discussions, the one that really stands out where they helped another user to create a workflow for [auto-commenting when opening a PR](https://github.com/orgs/community/discussions/69455).
- Sparkles ✨for @Laritello for nudging @alexpop-ads to the right direction and resolving an [issue related to the running a build](https://github.com/orgs/community/discussions/69293) on Vue web app.
- Kudos 👍  to our self-solver of the month @nedredmond - Thank You for coming back to the [discussion](https://github.com/orgs/community/discussions/67904) and sharing [how to call a composite action from another repository](https://github.com/orgs/community/discussions/67904). What an incredible self-solve 💯

### [Apple silicon powered M1 macOS larger runner for GitHub Actions](https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/)(Public Beta) 🆕 

⚡ Speed up your GitHub Actions jobs on macOS with an all new, faster Apple silicon powered M1 macOS larger runner for arm64. ⏩

Apple silicon (M1) hosted runners can now be used by any developer, team, or enterprise! You can try the new runners today by setting the `runs-on:` key to `macos-latest-xlarge` or `macos-13-xlarge`. The 12-core Intel macOS runner is still available as well and can be used by updating the `runs-on:` key to `macos-latest-large`, `macos-12-large`, or `macos-13-large` in your workflow file. Join the [Community Discussion](https://github.com/orgs/community/discussions/69211) to share thoughts and feedback.

You can track progress towards the general availability of macOS larger runners by following this [roadmap item](https://github.com/github/roadmap/issues/819). Also, macOS 14 will be available this year, and you can use this [roadmap item](https://github.com/github/roadmap/issues/813) to track progress towards it. To stay updated with the latest GitHub Actions updates, keep an eye  👀  on the [GitHub](https://github.blog/) blog.

### 👁️‍🗨️ [NODE_OPTIONS is now restricted from GITHUB_ENV](https://github.blog/changelog/2023-10-05-github-actions-node_options-is-now-restricted-from-github_env/) 🔐 

Due to security restrictions, users can no longer use GITHUB_ENV to set the NODE_OPTIONS environment variable in their workflows. Developers who have NODE_OPTIONS set as an environment variable will now receive an error: `Can't store NODE_OPTIONS output parameter using '$GITHUB_ENV' command.`

This change was introduced in [actions/runner v2.309.0](https://github.com/actions/runner/releases/tag/v2.309.0).  
For more information on how to set environment variables, please see our docs [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable).

### 🔒 [Secure deployment rollouts to protected environments based on select tag patterns](https://github.blog/changelog/2023-10-06-actions-secure-deployment-rollouts-to-protected-environments-based-on-select-tag-patterns/)

We now allow defining selected tag patterns for securing your deployments that can run against Actions environments.

Admins who want to have more secure and controlled deployments can now specify [selected tags or tag patterns](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#deployment-branches-and-tags) on their protected environments – Ex: You could now define that only deployments triggered by tags that match the pattern of "releases/*" could deploy to their "Production" environment.  
![Deployment Branches and Tags](https://i0.wp.com/user-images.githubusercontent.com/25389593/273319498-77aaed3b-9507-486e-9ff6-707e765efc90.png)

## 🗳️ Voice Your Opinions: Join Our Community Survey

 👂 Help us improve the GitHub Community: What's working and where can we make your experience better? Share now in our [Community Survey](https://github.com/orgs/community/discussions/69811) before October 25th!

## Level up 🆙  your GitHub Actions skillset with our learning pathways on automation.

Unlock 🔓  the full potential of GitHub with our specialized **learning pathways**. Each pathway is designed to help you master a new topic—automation, security, and administration & governance. With clear, actionable insights 📊  , you’ll refine your skills and elevate your projects. Whether enhancing development workflows, securing applications, or streamlining governance processes, these pathways are tailored to ensure you meet your needs efficiently and effectively.

## 🪐Universe 2023 is almost here: Are you Ready? 🔖 
- **[The GitHub Universe ‘23 agenda has landed](https://github.com/orgs/community/discussions/68412) 🚀** - We’re super excited 😮  for GitHub Universe and the countdown 📅  has begun. It is less than a month from now and we have the [list of agenda](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?utm_source=LinkedIn&utm_medium=Social&utm_campaign=schedule-launch) on our hands. You will have the chance to explore excited content sessions delivered by different speakers, across two days and three content tracks, all designed to level up ⬆️ your skills.   
-  **10 reasons (you never knew existed) about why you should attend Universe**:  [a list in Discussion](https://github.com/orgs/community/discussions/66508) about why Universe in real life is going to be the place to be 🌉. 
-  **Universe 2023 Website**: Join us as we build the future together, [our site](https://githubuniverse.com/) has you covered with FAQs, tickets, and some of our featured speakers. If you haven’t signed up already - what are you waiting for? ⚡ 

## Good Reads 📚 

- [How GitHub uses GitHub Actions and Actions larger runners to build and test GitHub.com](https://github.blog/2023-09-26-how-github-uses-github-actions-and-actions-larger-runners-to-build-and-test-github-com/)
- Learn how to [build organization-wide governance](https://github.blog/2023-04-05-building-organization-wide-governance-and-re-use-for-ci-cd-and-automation-with-github-actions/) and re-use them for CI/CD and automation with GitHub Actions.
- Love ❤️ reading📚 ? Love GitHub! [Discover 🌐 ](https://github.com/orgs/community/discussions/categories/discover?discussions_q=category%3ADiscover+sort%3Atop) some of the top-notch content to consume right in the GitHub Community, where we share blogs, articles, best practices, and tips & tricks from GitHub employees and users.
  - Learn more about the [scalable Continuous Integration (CI) patterns](https://github.com/orgs/community/discussions/50726) that can help your team grow significantly.
  - Read 📖  this [masterpiece](https://github.com/orgs/community/discussions/41489) from [@ayodejiayodele](https://github.com/ayodejiayodele) where he has explained some of the common pitfalls that often plague DevOps teams or organization.

💭  How are you planning to join us at the [Universe](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=schedule-launch) next month? In-person or Virtually? Share it in the comments below.

**Comment by jeremytchang** [comment]
> Due to security restrictions, users can no longer use GITHUB_ENV to set the NODE_OPTIONS environment variable in their workflows. Developers who have NODE_OPTIONS set as an environment variable will now receive an error: Can't store NODE_OPTIONS output parameter using '$GITHUB_ENV' command.

Do you have a workaround or do you accept the breakage of workflows?
